sbol_utilities/conversion.py

```diff
@@ -395,7 +396,17 @@ def command_line_converter(args_dict: Dict[str, Any]):
     if input_file_type == 'FASTA':
         doc3 = convert_from_fasta(input_file, namespace)
     elif input_file_type == 'GenBank':
-        doc3 = convert_from_genbank(input_file, namespace, args_dict['allow_genbank_online'])
+        allow_legacy_converter = args_dict['allow_legacy_converter']
+        converter = GenBank_SBOL3_Converter()
+        doc3 = converter.convert_genbank_to_sbol3(gb_file=input_file, sbol3_file=output_file, 
+                                                  namespace=namespace, write=False)
+        if not doc3:
+            logging.error(f"No SBOL3 output document was produced by new converter;\n    Please make sure you have the latest pip package installed.")
+            if allow_legacy_converter:
+                logging.info(f"Error encountered with conversion using new offline converter, reverting to legacy online converter.")
+                doc3 = convert_from_genbank(input_file, namespace, args_dict['allow_genbank_online'])
```

Comment:
The switching logic should happen _inside_ this call, not here, in order to support library usage.

Reply:
I have edited this, also I needed to add a check if returning doc3 is none while outputting (writing) the doc3 in the `command_line_converter` method.

Also, how should I go about writing tests for this?
Should I add final outputs from my converter into `test/test_files` and write test cases where the `--force-new-converter` is used?

Reply:
Yep, I have called the `convert_from_genbank()` directly in the unit test.